validation/poststop.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	tap "github.com/mndrix/tap-go"
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/runtime-tools/specerror"
+	"github.com/opencontainers/runtime-tools/validation/util"
+	uuid "github.com/satori/go.uuid"
+)
+
+func main() {
+	t := tap.New()
+	t.Header(0)
+
+	g := util.GetDefaultGenerator()
+
+	var output string
+	config := util.LifecycleConfig{
+		Actions: util.LifecycleActionCreate | util.LifecycleActionStart | util.LifecycleActionDelete,
+		PreCreate: func(r *util.Runtime) error {
+			r.SetID(uuid.NewV4().String())
+
+			output = filepath.Join(r.BundleDir, g.Spec().Root.Path, "output")
+			poststop := rspec.Hook{
+				Path: fmt.Sprintf("%s/%s/bin/sh", r.BundleDir, g.Spec().Root.Path),
+				Args: []string{
+					"sh", "-c", fmt.Sprintf("echo 'post-stop called' >> %s", output),
+				},
+			}
+
+			g.AddPostStopHook(poststop)
+			g.SetProcessArgs([]string{"sh", "-c", fmt.Sprintf("echo 'process called' >> %s", output)})
+			r.SetConfig(g)
+			return nil
+		},
+		PostCreate: func(r *util.Runtime) error {
+			outputData, err := ioutil.ReadFile(output)
+			if err != nil {
```

Comment:
Similar to 'poststart.go', we should get an error: the output file is not exist.